src/views/preview/presentation.jsx

```diff
@@ -3,26 +3,42 @@ const injectIntl = require('react-intl').injectIntl;
 const intlShape = require('react-intl').intlShape;
 const PropTypes = require('prop-types');
 const React = require('react');
+const Formsy = require('formsy-react').default;
 
+const sessionActions = require('../../redux/session.js');
+const decorateText = require('../../lib/decorate-text.jsx');
 const FlexRow = require('../../components/flex-row/flex-row.jsx');
 const Avatar = require('../../components/avatar/avatar.jsx');
 const CappedNumber = require('../../components/cappednumber/cappednumber.jsx');
 const placeholder = require('./gui-placeholder.png');
 const ShareBanner = require('../../components/share-banner/share-banner.jsx');
-const RemixList = require('../../components/remixlist/remixlist.jsx');
+const ThumbnailColumn = require('../../components/thumbnailcolumn/thumbnailcolumn.jsx');
+const InplaceInput = require('../../components/forms/inplace-input.jsx');
 
 require('./preview.scss');
 
 const PreviewPresentation = props => {
     const {
+        creditInfo,
+        editable,
+        faved,
+        favoriteCount,
         intl,
+        loved,
+        loveCount,
         projectInfo,
-        creditInfo,
-        remixes
+        remixes,
+        sessionStatus,
+        studios,
+        user,
+        onFavoriteClicked,
+        onLoveClicked,
+        onUpdate
         // ...otherProps TBD
     } = props;
     const shareDate = (projectInfo.history && projectInfo.history.shared) ? projectInfo.history.shared : '';
-    
+    const lovesClass = loved ? 'project-loves loved' : 'project-loves';
+    const favesClass = faved ? 'project-favorites favorited' : 'project-favorites';
```

Comment:
This is a good use for `classNames`: `classNames('project-loves', {loved: loved})` (and lends itself for using inline for the `className` prop).